src/test/scala/com/databricks/spark/avro/AvroSuite.scala

```diff
@@ -29,6 +29,15 @@ import org.scalatest.FunSuite
 /* Implicits */
 import TestSQLContext._
 
+private[avro] object DirectoryDeletion {
```

Comment:
No, not hacky -- this is pretty common for libraries, which have to minimize their dependencies. I would create an object called TestUtils and put this in there.